src/commands/promote.ts

```diff
@@ -153,14 +154,25 @@ export default class Promote extends Command {
       // copy debian artifacts
       const debArtifacts = [
         templateShortKey('deb', {bin: config.bin, versionShaRevision: debVersion(buildConfig), arch: 'amd64' as any}),
-        templateShortKey('deb', {bin: config.bin, versionShaRevision: debVersion(buildConfig), arch: 'i386' as any}),
         'Packages.gz',
         'Packages.xz',
         'Packages.bz2',
         'Release',
         'InRelease',
         'Release.gpg',
       ]
+      
+      // start
+      // TODO: remove in next major release
+      // node dropped 32-bit support for linux a long time ago, see:
+      // https://github.com/oclif/oclif/issues/770#issuecomment-1508719530
+      const arches = buildConfig.targets.filter(t => t.platform === 'linux')
+      
+      if (arches.find(a=> a.arch.includes('x86')) && lte(buildConfig.nodeVersion, 'v9.11.2')) {
```

Comment:
9.11.2 is the last known node version that shipped 32bit builds for linux:
https://nodejs.org/dist/latest-v9.x/

I was leaning just remove it from the debArtifacts push but maybe we could break someone?